mage_ai/data_preparation/shared/secrets.py

```diff
@@ -41,22 +73,17 @@ def create_secret(name: str, value: str):
     return secret
 
 
-def get_encryption_key() -> str:
-    secrets_dir = get_secrets_dir()
-    key_file = os.path.join(secrets_dir, 'key')
-
-    try:
-        with open(key_file, 'r') as f:
-            key = f.read()
-    except Exception:
-        key = None
-
-    return key
-
-
-def get_valid_secrets() -> List:
+def get_valid_secrets(
+    entity: Entity = Entity.GLOBAL,
+    pipeline_uuid: str = None,
+    project_uuid: str = None,
+) -> List:
     from mage_ai.orchestration.db.models.secrets import Secret
-    key = get_encryption_key()
+    key, _ = _get_encryption_key(
```

Comment:
Does this method still get secrets with the repo path?

Reply:
yes, I left it like this for now so that it still returns the same secrets that it did before this change. In the future, this method will need to be updated. I can add a comment